client/src/app/pages/past-seasons/past-seasons.component.ts

```diff
@@ -2,6 +2,7 @@ import { Component, OnInit } from '@angular/core';
 import { HistoryService } from '../../services/history.service';
 import { TimeService } from '../../services/time.service';
 import { FilterService } from '../../services/filter.service';
+import { orderBy } from 'lodash'
```

Comment:
I didn't just pull all of lodash into the minified .JS, did I?  Angular is smart enough to tree-shake and only include what it needs?  I didn't see an example of pulling in a single library on one of the Angular pages, please correct me if I did this wrong.